Adding JCR (JFrog Container Registry) Conformance Test

zendril opened the pull request:
You'll have to set the following secrets on your github:
          OCI_USERNAME: ${{ secrets.JCR_USERNAME }}
          OCI_PASSWORD: ${{ secrets.JCR_PASSWORD }}

I'll send separately.

zendril committed:
adding jcr workflow

zendril committed:
adding jcr workflow

zendril committed:
adding jcr workflow

zendril committed:
adding jcr workflow

zendril committed:
adding jcr workflow

zendril committed:
adding jcr workflow

zendril committed:
adding jcr workflow

zendril committed:
adding jcr workflow

zendril committed:
adding jcr workflow

zendril committed:
adding jcr workflow

zendril committed:
adding custom image build

zendril committed:
creating jcr-custom-image.yml

zendril committed:
adding custom image build

zendril committed:
adding custom image build

zendril committed:
adding custom image build

zendril committed:
adding custom image build

zendril committed:
adding custom image build

zendril committed:
adding custom image build

zendril committed:
adding custom image build

zendril committed:
adding custom image build

zendril committed:
adding custom image build

zendril committed:
adding custom image build

zendril committed:
adding custom image build

zendril committed:
adding custom image build

zendril committed:
adding custom image build

zendril committed:
adding custom image build

zendril committed:
adding custom image build

zendril committed:
adding custom image build

zendril committed:
adding custom image build

zendril committed:
adding custom image build

zendril committed:
working version, cleaned up

zendril committed:
updating the report to be jcr

zendril committed:
removing unused 8082 port

jdolitsky commented:
@zendril awesome!

2 things:
1. Can we change the top level dir to "config" vs. "configurations"
2. Can you add this to the table in readme?

zendril committed:
removing secrets, updating README.md

zendril committed:
renaming services to Commercial

zendril committed:
ABC, easy as 123

zendril commented:
> @zendril awesome!
> 
> 2 things:
> 
> 1. Can we change the top level dir to "config" vs. "configurations"

Done.

> 2. Can you add this to the table in readme?

Done. Renamed to Commercial and moved JFrog into here.

Also removed the secrets.
